config/spring.rb

```diff
@@ -0,0 +1,6 @@
+%w(
+  .ruby-version
+  .rbenv-vars
+  tmp/restart.txt
+  tmp/caching-dev.txt
+).each { |path| Spring.watch(path) }
```

Comment:
Not sure. Spring reloads the app in dev as things change? That seems a ok thing to leave in